docs/development-setup.md

```diff
@@ -52,3 +52,14 @@ More information about smoketests can be found [here](../smoketest/README.md).
 All unit tests can be run with `make test`.
 
 UI Unit tests are found under the directory of the file being tested, with the same file name, appended with `.test.js`. They can be run independently of the Go unit tests with `make jest`. Watch mode can be enabled with `make jest JEST_ARGS=--watch`.
+
+## Toolbox
+
+There is an abundance of free/open-source tooling out there, but these are some that work well for our team on mac at the time of writing
```

Comment:
```suggestion
There is an abundance of free/open-source tooling out there, but below are some tools that work well for our team:
```

I think we can leave out `on mac` and `at time of writing` since there will always be alternative ways to run these apps (VMs or similar), and the timestamp will show on the readme, so viewers should have a good sense if the doc gets outdated